book/src/5.in-depth-guides/5.ai/5.best-practices.md

```diff
@@ -0,0 +1,32 @@
+# Best Practices
+
+When sending messages using AI, Teams recommends a number of best practices to help with both user and developer experience.
+
+## AI-Generated Indicator
+
+When sending messages using AI, Teams recommends including an indicator that the message was generated by AI. This can be done by adding a `addAiGenerated` property to outgoing message. This will 
```

Comment:
Looks like you menat to add a little more?

Reply:
good catch!